templates/vue/src/components/NodeModal.vue

```diff
@@ -475,7 +475,10 @@ export default {
         this.updateNodeCoordinates()
         this.loading = true
 
-        if (this.node.mediaType === "url-embed" && this.node.behaviour !== "embed") {
+        if (
+          (this.node.mediaType === "url-embed" && this.node.behaviour !== "embed") ||
+          this.node.mediaFormat === "youtube"
+        ) {
```

Comment:
Can you change this to a computed property? Would be more readable given that it's used in multiple places.

Reply:
Done!